src/lib/network_peer/peered.ml

```diff
@@ -0,0 +1,5 @@
+(* peered.ml -- pair some data with a Peer.t *)
```

Comment:
can you call this `with_peer.ml` instead? This is the style we've been using in other places (see `with_hash`)

Reply:
code is gone